libs/contact/src/selectors/contact.selector.ts

```diff
@@ -0,0 +1,21 @@
+import { MemoizedSelector, createFeatureSelector, createSelector } from '@ngrx/store';
+import { DaffContactState } from '../reducers/contact.reducer';
+
+export interface State {
```

Comment:
I think this should be like "DaffContactReducerState" just so it's a little more clear what it is when it's imported.